documents/invoice/model_test.go

```diff
@@ -231,7 +231,7 @@ func TestInvoiceModel_calculateDataRoot(t *testing.T) {
 func TestInvoice_GenerateProofs(t *testing.T) {
 	i, err := createInvoice(t)
 	assert.Nil(t, err)
-	proof, err := i.CreateProofs([]string{"invoice.invoice_number", documents.CDTreePrefix + ".collaborators[0]", documents.CDTreePrefix + ".document_type"})
+	proof, err := i.CreateProofs([]string{"invoice.invoice_number", documents.CDTreePrefix + ".next_version", documents.CDTreePrefix + ".document_type"})
```

Comment:
Cant we create a test for a proof with the new collaborators under the read roles?

Reply:
will add that 👍